auto_invitation_to_notion/main.py

```diff
@@ -0,0 +1,127 @@
+import requests
+import copy
+from collections import defaultdict
+from typing import List, Dict
+
+from notion.client import NotionClient
+
+
+LUBYCON_PRIVATE_PAGE = '4594ead1-b975-451d-a578-2ffeea6aa452'
+LUBYCON_MENTOR_PAGE = '720e43a5-f572-440a-8e58-847b72359b16'
+LUBYCON_HUB_PAGE = 'd3ebe34b-34af-4b2f-b984-ae1e91cff7f3'
+
+
+def get_lubycon_admin_uid(client: NotionClient) -> str:
+    return client.get_email_uid().get('lubycon@gmail.com')
+
+
+def get_workspace_id(client: NotionClient, admin_uid: str) -> str:
+    response = client.post('getSpaces', {})
+    if response.ok:
+        workspace_id = list(response.json()[admin_uid]['space'].keys())[-1]
+        return workspace_id
+    else:
+        raise Exception(handle_request_error(response=response))
```

Comment:
요런 부분들은 Ealry Return 패턴으로 조금 더 가독성이 좋게 만들 수도 있을 것 같아유! 파이썬 잘 몰라서 문법은 틀릴 수도 있슴다 :pray:

```suggestion
    if response.ok == False:
        raise Exception(handle_request_error(response=response))
        
    workspace_id = list(response.json()[admin_uid]['space'].keys())[-1]
    return workspace_id
```

Reply:
네! 동일하게 작동합니다.
문법은 `if not response.ok: raise Exception(~~)`하는 형태로하면 좋을 것 같아서 그 부분만 감안해서 작업할게요!

Reply:
반영 완료했습니다.